src/engine/client/cl_main.cpp

```diff
@@ -754,14 +754,14 @@ void CL_FlushMemory()
 	if ( !com_sv_running->integer )
 	{
 		// clear the whole hunk
-		Hunk_Clear();
+		Hunk_ShutDownRandomStuffAndClear();
```

Comment:
heh, maybe rename it to ShutdownSubsystems() or something and don't have it call Hunk_Clear()... idk, this seems like a weird construct anyways.

Reply:
I'm hoping to do some more work on initialization and shutdown soon. In the meantime I like to give it a name that highlights the weirdness.